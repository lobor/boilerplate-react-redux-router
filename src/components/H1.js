import styled from 'styled-components'

module.exports = styled.h1`
  font-size: 4.0rem;
  font-weight: 300;
  line-height: 1.2;
  letter-spacing: -.1rem;
  @media (min-width: 550px){
    h1 {
        font-size: 5.0rem;
    }
  }
`;
