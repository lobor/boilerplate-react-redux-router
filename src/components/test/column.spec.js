import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import H1 from './../H1'

test('render column', () => {
  const tree = renderer.create(<H1 />).toJSON()
  expect(tree).toMatchSnapshot()
})
