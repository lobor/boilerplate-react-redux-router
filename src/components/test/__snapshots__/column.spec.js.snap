// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render column 1`] = `
.c0 {
  font-size: 4.0rem;
  font-weight: 300;
  line-height: 1.2;
  -webkit-letter-spacing: -.1rem;
  -moz-letter-spacing: -.1rem;
  -ms-letter-spacing: -.1rem;
  letter-spacing: -.1rem;
}

@media (min-width:550px) {
  .c0 h1 {
    font-size: 5.0rem;
  }
}

<h1
  className="c0"
/>
`;
